cmd/arena/commands/prune.go

```diff
@@ -79,17 +78,25 @@ func NewPruneCommand() *cobra.Command {
 					}
 				}
 			}
+			deleted := false
 			for _, job := range jobs {
 				if GetJobRealStatus(job) != "RUNNING" {
-					if job.Age() > (time.Duration(pruneArgs.days)*24*time.Hour + time.Duration(pruneArgs.hours)*time.Hour) {
-						deleteTrainingJob(job.Name())
+					if job.Age() > pruneArgs.since {
+						deleted = true
+						fmt.Printf("Delete %s %s", job.Trainer(), job.Name())
+						err = deleteTrainingJob(job.Name())
+						if err != nil {
+							fmt.Printf("Failed to delete %s %s, err: %++v", job.Trainer(), job.Name(), err)
+						}
 					}
 				}
 			}
+			if !deleted {
+				fmt.Println("No job need to be deleted")
+			}
 		},
 	}
 
-	command.Flags().Int64Var(&pruneArgs.days, "day",  10, "Specify clean job's days.")
-	command.Flags().Int64Var(&pruneArgs.hours, "hour", 0, "Specify clean job's hours.")
+	command.Flags().DurationVarP(&pruneArgs.since, "since","s",  24 * time.Hour, "Specify clean job's days.")
```

Comment:
1. we won't do any prune if the user doesn't specify  the relative duration for cleaning up.
2. Need clarify usage info. I think it's better to say "Only prune the jobs created before the given relative duration like 5s,2m, 3h or 10d"